### DISCUSSION TITLE
Customization ?

### DISCUSSION THREAD

**Discussion by BornTKill** [Status-Page/Status-Page#465] [Q&A] [answered]
Is there a way to theme the Statuspage ?

**Comment by herrtxbias** [comment, marked as answer]
Yes, you do have the possibility to theme your page.

You can add a "Config Revision" in the Admin Panel to set custom HTML in the header and / or footer, as well as custom CSS. You can also add custom JS in your header / footer HTML.

![image](https://cdn.herrtxbias.net/2023-02-17_11-12-49_6725c021-5b98-45e3-ac41-cbe7519c7b55.png)

**Comment by palicepta** [comment]
Hi all,

Did somebody try to customize the status page via custom styles?
Could you please provide some examples?

Thank you in advance.

**Comment by palicepta** [comment]
Hi all,

It would be nice to have a button to remove the dark mode change from the admin dashboard
